### PR TITLE
Create action creators

### PR DESCRIPTION
**Pull request by jpoist97**
Added action creators to complete redux setup. This doesn't change anything it just organizes the code to where I should've put it in the first place

### CODE REVIEW

File: components/Home/Home.jsx [hunk 1]
```diff
@@ -78,8 +71,8 @@ export default () => {
           name={cycleDetails && cycleDetails[cycles.selectedCycleIndex].name}
           subtext={cycleDetails && cycleDetails[cycles.selectedCycleIndex].muscleGroups}
           color={cycleDetails && cycleDetails[cycles.selectedCycleIndex].color}
-          leftPress={() => { dispatch({ type: DECREMENT_SELECTED_CYCLE_INDEX, cycleLength: cycleDetails.length }); }}
-          rightPress={() => { dispatch({ type: INCREMENT_SELECTED_CYCLE_INDEX, cycleLength: cycleDetails.length }); }}
+          leftPress={() => { dispatch(actions.cycles.decrementSelectedCycleIndex(cycleDetails.length)); }}
+          rightPress={() => { dispatch(actions.cycles.incrementSelectedCycleIndex(cycleDetails.length)); }}
           id={cycleDetails && selectedCycle.workouts[cycles.selectedCycleIndex]}
         />
         <WorkoutSwipeList items={workoutList} style={{ marginLeft: '10%' }} />
```
Comment (ctperry0301) [inline components/Home/Home.jsx:173]: The only redux-y thing that doesnt seem perfect yet is the recent workouts on the home page. If you finish a workout, I feel like it should be put at the front, and also the n/a days ago is a bit odd. 
Screen Shot 2021-01-12 at 1 34 13 PM

Also, ^^thats not something that needs to be addressed in this branch, I just stumbled across it during review.

**Review by ctperry0301 — DISMISSED** [review]
See comment, but looks good to go ahead and merge

File: components/Home/Home.jsx (hunk 1 above)
Comment (jpoist97) [reply components/Home/Home.jsx:173]: Yeah that's something that should be a part of Rohith's task updating Home to use Redux stuff

Comment (rohithdara) [reply components/Home/Home.jsx:173]: this is complete!

**Review by rohithdara — APPROVED** [review]
Looks good but I'm just confused as to why the refactoring was necessary -- just good practices?

**Comment by jpoist97** [comment]
> Looks good but I'm just confused as to why the refactoring was necessary -- just good practices?

Yep, pretty much. This is how you're supposed to set it up, but I forgot to last time I did it.

**Pull request merged**